tests/ui-toml/large_futures/clippy.toml

```
@@ -0,0 +1 @@
+future-size-threshold = 16384
```

```suggestion
future-size-threshold = 16384

```

These files are sadly now formatted automatically.

I think for readability, it would also be easier, if the test config value would be more distinct from the 16kb. Maybe 1024 would be better. But keeping the current size is a also fine :)